app/assets/javascripts/application.js

```diff
@@ -1,15 +0,0 @@
-// This is a manifest file that'll be compiled into application.js, which will include all the files
```

Comment:
this is not needed at all, sprockets initialization is done in `base.js.erb`

https://github.com/forem/forem/blob/c35f1c545d031722f3f36bf02f40e9a8d02e2bc2/app/assets/javascripts/base.js.erb#L1-L7